consensus/consensus-types/src/pipelined_block.rs

```diff
@@ -120,6 +120,58 @@ pub struct PipelineInputRx {
     pub commit_proof_fut: TaskFuture<LedgerInfoWithSignatures>,
 }
 
+/// A window of blocks that are needed for execution with the execution pool, EXCLUDING the current block
+#[derive(Clone)]
+pub struct OrderedBlockWindow {
```

Comment:
High level question: what's the expected behavior of handling blockmetadata/statecheckpoint/blockepilogue txns? Can you double check to see if the downstream change is needed, and document it somewhere?